stats/handlers.go

```diff
@@ -38,6 +38,14 @@ type RPCTagInfo struct {
 	// FailFast indicates if this RPC is failfast.
 	// This field is only valid on client side, it's always false on server side.
 	FailFast bool
+	// NameResolutionDelay indicates whether there was a delay in name
+	// resolution.
+	//
+	// This field is only valid on client side, it's always false on server side.
+	NameResolutionDelay bool
+	// IsTransparentRetry indicates whether the stream is undergoing a
+	// transparent retry.
+	IsTransparentRetry bool
```

Comment:
this is also valid on client side only?

Also, it looks like its already being passed to HandleRPC https://github.com/grpc/grpc-go/blob/master/stream.go#L428. Do we have to track it still? We only need to add it while populating span which can be done directly under begin case

Reply:
Yeah this was already handled earlier, but somehow while rebasing with my earlier commits, this came. In the populateSpan, we were already using from Begin handler.